DrawableAnimations/README.md

```diff
@@ -0,0 +1,49 @@
+Android DrawableAnimations Sample
+=================================
+
+A collection of demos for drawable animation APIs.
+
+<img src="screenshots/ss-home.png" height="400" alt="Home screen">
```

Comment:
Would be good if these were GIFs, since this is showing an animation

Reply:
Done.